R/translate-sql-helpers.R

```diff
@@ -132,19 +132,62 @@ copy_env <- function(from, to = NULL, parent = parent.env(from)) {
 #' @param pad If `TRUE`, the default, pad the infix operator with spaces.
 #' @export
 sql_infix <- function(f, pad = TRUE) {
+  # (x1 <- expr((2 - 1) * x))
+  # #> (2 - 1) * x
+  #
+  # (x2 <- expr(!!expr(2 - 1) * x))
+  # #> (2 - 1) * x
+  #
+  # `x1` and `x2` look the same when printed but their AST (abstract syntax tree)
```

Comment:
You could point to https://adv-r.hadley.nz/quasiquotation.html#non-standard-ast to keep this comment a bit shorter.